microsoft/testsuites/display/drm_sanity.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.base_tools import Cat
+from lisa.tools import Lsmod
+
+@TestSuiteMetadata(
+    area="drm",
+    category="functional",
+    description="""
+    This test suite uses to verify drm driver sanity.
+    """,
+    requirement=simple_requirement(unsupported_os=[]),
```

Comment:
1. If a test case can be run on Windows, this line is needed. Otherwise, it doesn't.
2. It looks this test case applies to Azure only, please add platform restriction.

Reply:
yes this is only for linux, will add restructions